Templates/CSharp/UWP/ProjectTemplates/CSharp/Windows UAP/UWP/UnitTestAppMainPage.xaml

```diff
@@ -0,0 +1,27 @@
+﻿<Page
+    x:Class="$safeprojectname$.UnitTestAppMainPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:$safeprojectname$"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+    <Page.Resources>
+        <Style x:Key="CustomStyle" TargetType="TextBlock">
+            <Setter Property="FontSize" Value="20"/>
+            <Setter Property="Width" Value="150"/>
```

Comment:
I would just remove all Width, Height, HA and VA. They're already set on the StackPanel.